test/scalability/runner/main.go

```diff
@@ -170,10 +186,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *metricsScrapeInterval != 0 && *metricsScrapeURL != "" {
+		dumpTar := path.Join(*outputDir, "metricsDump.tgz")
+		wg.Add(1)
```

Comment:
why this one? The routine that runScraper launches already increments the wg.

Reply:
done